packages/create-cloudflare/readme.md

```diff
@@ -0,0 +1,70 @@
+# create-cloudflare
+
+> Create new Cloudflare projects with one command
+
+> **Note**
+> This package has now been sunsetted! We recommend using the `wrangler generate [name] [template]` command to create new projects with templates.
```

Comment:
If we have sunsetted this package, perhaps we should not bring it in to this monorepo?

Reply:
If we are no longer implementing it in generate, just using our own code than I agree with this and we can avoid the unnecessary maintenance burden.

Reply:
Removed the package!